spec/features/navigation_spec.rb

```diff
@@ -32,6 +56,30 @@
         expect(page).to have_content("Cart: 0")
       end
 
+      visit '/home'
```

Comment:
this all blew my mind good job 
![](https://media.giphy.com/media/WfIDKyIzU95kY/giphy.gif)